devito/ops/operator.py

```diff
@@ -33,32 +31,20 @@ def _specialize_iet(self, iet, **kwargs):
 
         ops_block = OpsBlock('block')
 
-        # Extract all symbols that need to be converted to ops_dat
         dims = []
-        to_dat = set()
         for section, trees in find_affine_trees(iet).items():
             dims.append(len(trees[0].dimensions))
-            symbols = set(FindSymbols('symbolics').visit(trees[0].root))
-            symbols -= set(FindSymbols('defines').visit(trees[0].root))
-            to_dat |= symbols
 
-        # To ensure deterministic code generation we order the datasets to
-        # be generated (since a set is an unordered collection)
-        to_dat = filter_sorted(to_dat)
-
-        name_to_ops_dat = {}
         pre_time_loop = []
-        for f in to_dat:
-            if f.is_Constant:
-                continue
-
-            pre_time_loop.extend(create_ops_dat(f, name_to_ops_dat, ops_block))
```

Comment:
oops, didn't see you removed this. This should stay here as per my other comment